packages/expo-cli/src/commands/export.js

```diff
@@ -76,8 +168,10 @@ export default (program: any) => {
     )
     .option('-d, --dump-assetmap', 'Dump the asset map for further processing.')
     .option('--dev', 'Configures static files for developing locally using a non-https server')
-    .option('--s, --dump-sourcemap', 'Dump the source map for debugging the JS bundle.')
+    .option('-s, --dump-sourcemap', 'Dump the source map for debugging the JS bundle.')
     .option('-q, --quiet', 'Suppress verbose output from the React Native packager.')
+    .option('--merge-src-dir [dir]', 'A repeatable source dir to merge in.', collect, [])
```

Comment:
Is this command documented more thoroughly somewhere? I wonder if people will understand what "a repeatable source dir" means.

Reply:
i've got a sibling docs pr in universe: https://github.com/expo/universe/pull/3139